include/zephyr/bluetooth/services/ots.h

```diff
@@ -888,6 +901,20 @@ struct bt_ots_client_cb {
 	 */
 	void (*obj_data_written)(struct bt_ots_client *ots_inst,
 				 struct bt_conn *conn, size_t len);
+
+	/** @brief Callback function when checksum indication is received.
+	 *
+	 *  Called when the oacp_ind_handler received response of
+	 *  OP BT_GATT_OTS_OACP_PROC_CHECKSUM_CALC.
+	 *
+	 *  @param ots_inst          Pointer to the OTC instance.
+	 *  @param conn              The connection to the peer device.
+	 *  @param err               Error code (bt_ots_olcp_res_code).
+	 *  @param crc               Checksum code (bt_ots_olcp_res_code).
+	 */
+	void (*obj_checksum_got)(struct bt_ots_client *ots_inst,
+				 struct bt_conn *conn, int err, uint32_t crc);
```

Comment:
Is `crc` the best name for the value? Wouldn't it make more sense to just use `checksum`?

Reply:
Will revise.

Reply:
Change to ``checksum``.